documentation/modules/configuring/proc-config-mirrormaker2-securing-connection.adoc

```diff
@@ -0,0 +1,257 @@
+
+The examples also show the ACLs needed by MirrorMaker 2.0 to allow operations on the source and target Kafka clusters.
+
+.Prerequisites
+
+* Strimzi is running
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Configure two `Kafka` resources, one to secure the source Kafka cluster and one to secure the target Kafka cluster.
++
+You can add listener configuration for authentication and enable authorization.
++
+In this example, an internal listener is configured for a Kafka cluster with TLS encryption and authentication.
+Kafka `simple` authorization is enabled.
++
+.Example source Kafka cluster configuration with TLS authentication
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-source-cluster
+spec:
+  kafka:
+    version: {DefaultKafkaVersion}
+    replicas: 1
+    listeners:
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+        authentication:
+          type: tls
+    authorization:
+      type: simple
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      default.replication.factor: 1
+      min.insync.replicas: 1
+      inter.broker.protocol.version: "{DefaultInterBrokerVersion}"
+    storage:
+      type: jbod
+      volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+  zookeeper:
+    replicas: 1
+    storage:
+      type: persistent-claim
+      size: 100Gi
+      deleteClaim: false
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
+----
+
+. Create or update the `Kafka` resources.
++
+[source,shell,subs=+quotes]
+----
+kubectl apply -f _<kafka_configuration_file>_
+----
++
+The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication within the Kafka cluster.
++
+The certificates are created with the same name as the Kafka resource.
+
+. Configure two `KafkaUser` resources, one for a user of the source Kafka cluster and one for a user of the target Kafka cluster.
++
+The users provide the credentials required for MirrorMaker to connect to the source and target Kafka clusters.
++
+--
+.. Configure the same authentication and authorization types as the corresponding source and target Kafka cluster.
+For example, if you used `tls` authentication and `simple` authorization type in the `Kafka` configuration for the source Kafka cluster,
+use the same in the `KafkaUser` configuration.
+
+.. Configure the ACLs needed by MirrorMaker 2.0 to allow operations on the source and target Kafka clusters.
++
+The ACLs are used by the internal MirrorMaker connectors, and by the underlying Kafka Connect framework.
+--
++
+.Example user configuration for TLS client authentication
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaUserApiVersion}
```

Comment:
Same as above, the users are actually different. So listing only one is weird and can be confusing. I would either list none or both.

Reply:
I've added both